docs/DafnyRef/UserGuide.md

```diff
@@ -617,9 +618,15 @@ enforce-determinism = true
 warn-shadowing = true
 ```
 
-Under the section `[options]`, any options from the Dafny CLI can be specified using the option's name without the `--` prefix. When executing a `dafny` command using a project file, any options specified in the file that can be applied to the command, will be. Options that can't be applied or are misspelled, are ignored.
-
-When using a Dafny IDE based on the `dafny server` command, the IDE will search for project files by traversing up the file tree looking for the closest `dfyconfig.toml` file it can find. Options from the project file will override options passed to `dafny server`.
+- At most one `.toml` file may be named on the command-line; when using the command-line no `.toml` file is used by default.
```

Comment:
> At most one `.toml` file may be named on the command-line

Is that good to mention in the documentation? I will also find out if I try it on the command line. I feel 99% of users won't try this and are just slowed down by larger documentation, while the 1% that doesn't attempt to do this because of this documentation only has a minor benefit.

Reply:
Its a reference manual, not a tutorial -- so my expectations are that the document defines the behavior.
Reasonable alternate behavior is that later things on the command-line supersede earlier ones, as options do.